text/text.md

```diff
@@ -72,50 +72,57 @@ OK, jungle. And you tell tall tales around the fire, right?
 it happened.  Yes, sir, I'm sure, what are you, a fuckin' people's court?  
 Do I have to take the oath or what?
 
 Alright, then. This is the real story of how I came to be in the
 possession of Manuel the Man-Made Man. It starts in Mobile, which is
-only fitting, given the circus obsession with phrases with words that
-all start the same. Mobile, yes, time ago it was in Alabama, but now
-it is a rough and tumble harbor town close to the border of the United
+only fitting, given the circus obsession with phrases that
+all start the same.  Ah, Mobile. Long ago it was in Alabama, but now
+it is a rough-and-tumble harbor town close to the border of the United
 All Races People's Republic of Florida. That's a pretty long name for
 a place that starts at Pascagoula in the West and and goes no further
-than Brunswick in the East, if only our people have their way. Which
+than Brunswick in the East, <i>if only our people have their way. Which
 they will, I have no doubt as a loyal citizen of the Republic.  Every
 dawning day brings new brothers to our new republic. Brothers of all
-races, creeds and dispositions.
-
+races, creeds and dispositions.</i>
+#####The last part where I italicized is a bit confusing.  What people? What way? I think if you maybe explained it a little more or put a transition here (like a sentence), it would make the part less confusing.
 OK, you all know that, no hooting, on with the story. But you all
-reckon now I'm a loyal Republic citizen, ain't I? 
+reckon now I'm a loyal Republic citizen, right? 
 
 So. It was early winter, but the weather was still sunny and mild. I was walking
-along the docks, as I was wont to do. The air was clear and I could see ships
-docking and leaving, ships from all our sister nations of the world, Spain, Cuba, 
-Colombia, Puerto Rico, but also from not-so-friendly places 
-who decided the buck needn't stop here and had resumed trade as soon as the
+along the docks, as I like to do. The air was clear, and I could see ships
+docking and leaving, ships from all our sister nations of the world, like Spain, Cuba, 
+Colombia, and Puerto Rico. There were also some from not-so-friendly places 
+who decided the buck #Not sure what the buck is? needn't stop here and had resumed trade as soon as the
 smoke from the Freedom War settled down and become the smoke of the
-smokestacks built during the Planazo. Formerly unfriendly places like  
-the imperialist United Kingdom, the Austrohungarian Empire, those bohunks... The
-air swarmed also with the airships about to dock or leave from the
-airship port, the Spanish clover-fronted ones, the elegant Cubans,
-even the odd bulky and slumbering American, you could just spend the
+smokestacks built during the Planazo. 
+####This is a run-on sentence, but I'm not sure what the best way to divide this is. 
+Formerly unfriendly places like the imperialist United Kingdom, the Austrohungarian Empire, those bohunks... 
+####What is a bohunk?
```

Comment:
Whitey, cracker jack, derogatory term directed to white people, specially from Europe, specially from the austro-hungarian empire http://www.urbandictionary.com/define.php?term=Bohunk